client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-category-list.tsx

```diff
@@ -43,9 +43,13 @@ const ScreenCategoryList = ( {
 	const categoriesInOrder = useCategoriesOrder( categories );
 
 	const handleFocusOutside = ( event: Event ) => {
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

ChatGPT suggested the code below with named variables and early return 😌

```ts
const handleFocusOutside = (event: Event) => {
  const target = event.target as HTMLElement;

  // Click outside the sidebar or action bar to close Pattern List
  const isInPatternActionBar = target.closest('.pattern-action-bar');
  const isInPatternAssemblerSidebar = target.closest('.pattern-assembler__sidebar');

  if (isInPatternActionBar || isInPatternAssemblerSidebar) {
    return;
  }

  setSelectedCategory(null);
  onTogglePatternPanelList?.(false);
};
```

Reply:
I'd prefer to early return when the rest part are bigger enough 😆